src/workers/cdc.ts

```diff
@@ -193,6 +195,14 @@ const onCommentChange = async (
         data.payload.after.id,
       );
     }
+  } else if (data.payload.op === 'u') {
+    await notifyCommentEdited(logger, data.payload.after);
```

Comment:
Let's check here that it was the content that got edited

Reply:
The only thing that can change from here is the `content` (and `contentHtml`) but I agree on making it explicit just to be sure 🚢